src/appleseed/renderer/modeling/bssrdf/sss.cpp

```diff
@@ -112,7 +112,38 @@ double reduced_extinction_coefficient(
     const double    dmfp,
     const double    alpha_prime)
 {
-    return 1.0 / (sqrt(3.0 * (1.0 - alpha_prime)) * dmfp);
+    const double sigma_tr = 1.0 / dmfp;
```

Comment:
The original expression was more efficient (one less divide). Maybe simply add a comment saying that 1.0 / dmfp is sigma_tr?